packages/clerk-js/src/ui/components/OrganizationList/OrganizationListPage.tsx

```diff
@@ -79,17 +79,17 @@ export const OrganizationListPage = withCardStateProvider(() => {
   );
 });
 
-const OrganizationListFlows = ({ showListInitially }: { showListInitially: boolean }) => {
-  const environment = useEnvironment();
-  const { navigateAfterSelectOrganization, skipInvitationScreen } = useOrganizationListContext();
-  const [isCreateOrganizationFlow, setCreateOrganizationFlow] = useState(!showListInitially);
-  return (
-    <>
-      {!isCreateOrganizationFlow && (
-        <OrganizationListPageList onCreateOrganizationClick={() => setCreateOrganizationFlow(true)} />
-      )}
-
-      {isCreateOrganizationFlow && (
+export const OrgListCreateOrganizationPage = withCardStateProvider(
+  (props: {
+    setCreateOrganizationFlow: React.Dispatch<React.SetStateAction<boolean>>;
+    isCreateOrganizationFlow: boolean;
+    showListInitially: boolean;
+  }) => {
+    const { showListInitially, isCreateOrganizationFlow, setCreateOrganizationFlow } = props;
+    const environment = useEnvironment();
+    const { navigateAfterSelectOrganization, skipInvitationScreen } = useOrganizationListContext();
+    return (
+      <>
```

Comment:
Github makes this hard to read. Basically i wrapped each flow with their own card provider.